packages/react-components/react-datepicker-compat/src/components/CalendarDayGrid/useWeekCornerStyles.ts

```diff
@@ -88,11 +84,8 @@ export function useWeekCornerStyles(props: CalendarDayGridProps) {
             day.isSelected,
           );
 
-        const style = [];
-        style.push(calculateRoundedStyles(above, below, left, right));
-        style.push(calculateBorderStyles(above, below, left, right));
-
-        weekCornersStyled[weekIndex + '_' + dayIndex] = style.join(' ');
+        const style = mergeClasses(calculateRoundedStyles(above, below, left, right));
+        weekCornersStyled[weekIndex + '_' + dayIndex] = style;
```

Comment:
I think you should be fine directly applying the returned value from `calculateRoundedStyles` since you're already calling `mergeClasses` at the end of it:

```suggestion
        weekCornersStyled[weekIndex + '_' + dayIndex] = calculateRoundedStyles(above, below, left, right);
```